lib/main.gi

```diff
@@ -2713,3 +2713,32 @@ end);
 ##
 #E
 ##
+
+IsMaximal:=function(N,A,D)
+The function IsMaximal checks whether A/D is maximal abelian subgroup of N/D 
+
+----------------------------
+
+IsMaximal:=function(N,A,D)

```

Comment:
Apparently lines above are supposed to be comments? I suggest to use the same formatting style like in the library:

```
#############################################################################
##
## IsMaximal(N,A,D)
##
## The function IsMaximal checks whether A/D is maximal abelian subgroup of N/D
##
```
and do not use the separating line of `---------------------------`.